addons/l10n_in_stock/models/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _l10n_in_compute_ewaybill_price_unit(self):
+        self.ensure_one()
+        return self.product_id.uom_id._compute_price(
+            self.product_id.with_company(self.company_id).standard_price, self.product_uom
+        )
+
+    def _l10n_in_compute_line_tax(self):
+        self.ensure_one()
+        taxes = (
+            self.picking_code == "incoming" and
+            self.product_id.supplier_taxes_id or self.product_id.taxes_id
+        )
+        return self._get_l10n_in_fiscal_position(
```

Comment:
@jco-odoo I m not sure we should have a reference to a method before it is defined?

Reply:
@hamo-odoo You can not just move it to l10n_in_stock?

Reply:
Hmmm but I guess we need to recompute two partner_ids from ewaybill here to checkout the transaction type 🤔 at least that would look valid here

Reply:
@jco-odoo had a word with @jva-odoo he suggested that we use fiscal_position instead of transaction_type please check the last commit (fiscal might not be that helpful now but after the auto fiscal merges it will be an ideal thing)